demo/standalone/Vagrantfile

```diff
@@ -38,7 +38,7 @@ if private_build
   aci_location = "/release/quay.io-rook-rookd-dev.aci"
   aci_options = "--insecure-options=image"
 else
-  aci_location = "quay.io/rook/rookd:latest"
+  aci_location = "quay.io/rook/rookd:master-latest"
```

Comment:
tests under `e2e` have a known instance of `latest` that they are changing to `master-latest`, but I can't find any other instances of `:latest`